slsim/image_simulation.py

```diff
@@ -78,9 +79,28 @@ def sharp_image(
     :param num_pix: number of pixels per axis
     :param with_source: bool, if True computes source
     :param with_deflector: bool, if True includes deflector light
+    :param angle_rot: rotation angle of the coordinate system
     :return: 2d array unblurred image
     """
     kwargs_model, kwargs_params = lens_class.lenstronomy_kwargs(band)
+    rotation_angle = (
+        angle_rot * np.pi / 180
+    )  # rotation angle in North to West in radian
+    scale1_1 = -delta_pix * np.cos(rotation_angle)
+    scale1_2 = delta_pix * np.sin(rotation_angle)
+    scale2_1 = delta_pix * np.sin(rotation_angle)
+    scale2_2 = delta_pix * np.cos(rotation_angle)
+    ra_at_xy_0_original = (num_pix - 1) / 2
+    dec_at_xy_0_original = (num_pix - 1) / 2
+    pix2angle_transform_rot = np.array([[scale1_1, scale1_2], [scale2_1, scale2_2]])
+    ra_at_xy_0, dec_at_xy_0 = pix2angle_transform_rot.dot(
+        [-ra_at_xy_0_original, -dec_at_xy_0_original]
+    ).T
+    kwargs_pixel_grid = {
```

Comment:
Is the issue that the coordinate system is not aligned between DC2 and SLSim or that the angle conventions in the parameters are not aligned? I would not mix the two and make sure that the coordinate system is identical to the definitions in DC2 and instead make sure the parameter definitions are changed. The way you implemented it can lead to confusion when the user also inputs other parts